src/web_interface/components/jinja_filter.py

```diff
@@ -215,11 +215,11 @@ def _setup_filters(self):  # noqa: PLR0915
         self._app.jinja_env.filters['nice_unix_time'] = flt.nice_unix_time
         self._app.jinja_env.filters['nice_virtual_path_list'] = self._nice_virtual_path_list
         self._app.jinja_env.filters['number_format'] = flt.byte_number_filter
```

Comment:
`octal_to_readable` is still a valuable filter. Do not remove it.

```suggestion
        self._app.jinja_env.filters['number_format'] = flt.byte_number_filter
        self._app.jinja_env.filters['octal_to_readable'] = flt.octal_to_readable
```